packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -274,36 +274,32 @@ export async function willForceUpload(
             )
         );
     }
-    vscode.window
-        .showInformationMessage(
-            localize("saveFile.info.confirmUpload", "Would you like to overwrite the remote file?"),
-            localize("saveFile.overwriteConfirmation.yes", "Yes"),
-            localize("saveFile.overwriteConfirmation.no", "No")
-        )
-        .then(async (selection) => {
-            if (selection === localize("saveFile.overwriteConfirmation.yes", "Yes")) {
-                const uploadResponse = Gui.withProgress(
-                    {
-                        location: vscode.ProgressLocation.Notification,
-                        title,
-                    },
-                    () => {
-                        return uploadContent(node, doc, remotePath, profile, binary, null, returnEtag);
+    Gui.infoMessage(localize("saveFile.info.confirmUpload", "Would you like to overwrite the remote file?"), {
+        items: [localize("saveFile.overwriteConfirmation.yes", "Yes"), localize("saveFile.overwriteConfirmation.no", "No")],
+    }).then(async (selection) => {
+        if (selection === localize("saveFile.overwriteConfirmation.yes", "Yes")) {
+            const uploadResponse = Gui.withProgress(
+                {
+                    location: vscode.ProgressLocation.Notification,
+                    title,
+                },
+                () => {
+                    return uploadContent(node, doc, remotePath, profile, binary, null, returnEtag);
+                }
+            );
+            uploadResponse.then((response) => {
```

Comment:
Same comment as before regarding then vs await.
Not sure if there is a preference, but I rather not mix them if possible 😋